pulpcore/cli/common/generic.py

```diff
@@ -105,9 +110,6 @@ def __init__(
     def echo(self, message: str, nl: bool = True, err: bool = False) -> None:
         click.echo(message, nl=nl, err=err)
 
-    def prompt(self, text: str, hide_input: bool = False) -> t.Any:
```

Comment:
I assume this was a never-used (and kind of pointless) method?

Reply:
Let's say this change pushes the ask-me-for-a-password one abstraction level down where we can use `click.promt` directly.